docusaurus.config.js

```diff
@@ -305,6 +305,12 @@ module.exports = {
               "/administration/private-repositories/ssh-key/",
             ],
           },
+          {
+            to: "manifest/index/",
```

Comment:
can I move this to the Netlify redirects config file? any preference between both files?